internal/olm/fbcutil/util.go

```diff
@@ -169,6 +171,14 @@ func RenderRefs(ctx context.Context, refs []string, skipTLSVerify bool, useHTTP
 	return declcfg, nil
 }
 
+func dirNameFromRefs(refs []string) string {
+	dirNameBytes := []byte(strings.ReplaceAll(strings.Join(refs, "_"), "/", "-"))
+	hash := sha256.New()
+	hash.Write(dirNameBytes)
+	hashBytes := hash.Sum(nil)
+	return fmt.Sprintf("%x", hashBytes)
```

Comment:
Can we shorten this to something like:
```suggestion
	return string(hash.Sum(nil))
```
?

If not this is fine, but I think those are equivalent and doesn't require use of the `fmt.Sprintf` function

Reply:
Thanks for the comment @everettraven - but these two forms are not equivalent. `string(hash.Sum(nil))` will try to use the random bytes as characters. The result is unexpected - we'll end up with wrong UTF-8 chars, non-printable chars and so on. 

The original format generates a hex string (2 hex digit per byte).